jsdoc.conf.json

```diff
@@ -5,7 +5,8 @@
   },
   "mocha-docdash": {
     "sort": true,
-    "static": false
+    "static": false,
+    "headScripts": ["/js/matomo.js"]
```

Comment:
same question from above

Reply:
This is necessary because these are in `/api/`, and the js is in `/js/`, not `/api/js/`.